src/main/java/uk/gov/companieshouse/api/accounts/exception/handler/GlobalExceptionHandler.java

```diff
@@ -35,6 +37,16 @@
     @Value("${invalid.value}")
     public String invalidValue;
 
+    @ExceptionHandler(value = {Exception.class})
```

Comment:
No need for the explicit `value` or curly braces. The braces are used if you're capturing more than one type of exception. So this line can become `@ExceptionHandler(Exception.class)`